release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1014,6 +1014,24 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
+
+* Before this update, the package server was not aware of pod topology when defining its leader election duration, renewal deadline, and retry periods. As a result, the package server strained topologies with limited resources, such as single node environments. This update introduces a `leaderElection` package that sets reasonable lease duration, renewal deadlines, and retry periods. This fix reduces strain on clusters with limited resources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048563[*BZ#2048563*])
+
+* Previously, there was a bad catalog source in the openshift-marketplace. Because of this, all subscriptions were blocked. With this update, if there is a bad catalog source in the openshift-marketplace, users can subscribe to an operator from a quality catalog source of their own namespace with the OG annotation. As a result, if there is a bad catalog source in the local namespace, the user cannot subscribe to any operator in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076323[*BZ2076323*]
+
+* Previously, info level logs were being generated during operator-marketplace polling, which caused log spam. This update uses the command line flag to reduce the log line to the debug level and adds more control of the log levels for the user. As a result, this reduces log spam. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057558[*BZ#2057558*]
+
+* Previously, each component managed by CVO consists of yamls defined in the `/manifest` directory in the root of a project's repo. When removing a yaml from the `/manifest` directory, you had to add the `release.openshift.io/delete: “true”` annotation, otherwise the CVO would not delete the resources from the cluster. This update reintroduces any resources that were removed from the `/manifest` and includes the `release.openshift.io/delete: “true”` annotation so that the CVO cleans up the resources. As a result, resources that are no longer required for the OLM component are removed from the customers' clusters.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1975543[*BZ#1975543*])
+
+* Previously, the CheckRegistryServer function used by GRPC catalogSources did not confirm the existence of the serviceAccount associated with the catalogSource. This caused the existence of an unhealthy catalogSource with no serviceAccount. With this update, the GRPC CheckRegistryServer function checks if the serviceAccount exists, which will recreate the service if it's not found. As a result, the OLM will recreate serviceAccounts owned by GRPC catalogSouces if they do not exist. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975543[*BZ#1975543*])
```

Comment:
```suggestion
* Previously, the CheckRegistryServer function used by GRPC catalogSources did not confirm the existence of the serviceAccount associated with the catalogSource. This caused the existence of an unhealthy catalogSource with no serviceAccount. With this update, the GRPC CheckRegistryServer function checks if the serviceAccount exists, which will recreate the service if it is not found. As a result, the OLM will recreate serviceAccounts owned by GRPC catalogSouces if they do not exist. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975543[*BZ#1975543*])
```

Reply:
```suggestion
* Previously, the CheckRegistryServer function used by GRPC catalog sources did not confirm the existence of the service account associated with the catalog source. This caused the existence of an unhealthy catalog source with no service account. With this update, the GRPC `CheckRegistryServer` function checks if the service account exists and recreates the service if it is not found. As a result, the OLM recreats service accounts owned by GRPC catalog souces if they do not exist. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975543[*BZ#1975543*])
```

Please review the [API object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) in the contrib guide.